### ISSUE TITLE
field naming consistency in beacon/chainhead

### ISSUE THREAD

**Issue by tzapu** [prysmaticlabs/prysm#4067]
hello,

since you are actively looking to improve consistency, i figured i'd add this as well:
```
{
  "headBlockRoot": "vBbnABaFPpWY7ODgSsRnp/Qcfo78e+jxic8d/3KMsp0=",
  "headBlockSlot": "2430",
  "headBlockEpoch": "303",
  "finalizedBlockSlot": "2408",
  "finalizedEpoch": "301",
  "finalizedBlockRoot": "FYsJfe7jlKQzehCNq/76ryF2m585U0NZ1W/f9c4HVQg=",
  "justifiedBlockSlot": "2416",
  "justifiedEpoch": "302",
  "justifiedBlockRoot": "ksdUzZ5YZDcarWbmO9+eKJzk0CHsEch226xwKpV6rUA=",
  "previousJustifiedSlot": "2408",
  "previousJustifiedEpoch": "301",
  "previousJustifiedBlockRoot": "FYsJfe7jlKQzehCNq/76ryF2m585U0NZ1W/f9c4HVQg="
}
```
you have: headBlockSlot, finalizedBlockSlot, justifiedBlockSlot and then previousJustifiedSlot which seems to be missing a Block.
you have: finalizedEpoch, justifiedEpoch, previousJustifiedEpoch and then headBlockEpoch whcih seems to have and extra Block in the name.

if it was intentionally left like this, please feel free to close.

thanks for all your hard work

**Comment by terencechain** [comment]
let's renamed to the following:
```
* headSlot
* headEpoch
* headBlockRoot
* finalizedSlot
* finalizedEpoch
* finalizedBlockRoot
...
```

**Comment by prestonvanloon** [comment]
Sounds good to me

**Comment by rauljordan** [comment]
Closed by #59
